rest/src/main/java/com/epam/esm/rest/exception/handler/RestExceptionHandler.java

```diff
@@ -0,0 +1,68 @@
+package com.epam.esm.rest.exception.handler;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.HttpRequestMethodNotSupportedException;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
+
+import com.epam.esm.rest.exception.InvalidRequestParametersException;
+import com.epam.esm.rest.exception.NotFoundException;
+import com.epam.esm.service.exception.ServiceValidationException;
+
+@ControllerAdvice
+public class RestExceptionHandler {
+
+	@ExceptionHandler
+	public ResponseEntity<ErrorResponse> handleNotFoundException(NotFoundException exc) {
+		ErrorResponse error = new ErrorResponse(HttpStatus.NOT_FOUND.value(), exc.getMessage(),
+				System.currentTimeMillis());
+		return new ResponseEntity<>(error, HttpStatus.NOT_FOUND);
```

Comment:
You could specify HttpStatus in annotation, get rid of ResponseEntity and return ErrorResponse directly.